lib/mintel/dashboards/components/templates.libsonnet

```diff
@@ -145,10 +145,19 @@ local template = grafana.template;
     label='NS',
     hide=hide,
   ),
+  slo_operator_namespaces(current='', hide=''):: template.new(
+    'slo_namespace',
```

Comment:
Is this referring to the operator namespace? If so, there could be some confusion between the slo's (objective vs operator)

Reply:
no this is about the slo objects, i will rename it

Reply:
changed